MaxText/checkpointing.py

```diff
@@ -103,7 +103,7 @@ def create_orbax_emergency_checkpoint_manager(
 
   # Only create directories if running on GPUs as the previous
   # directory structure might be assumed by TPUs
-  if global_mesh.devices.flatten()[0].platform == 'gpu':
+  if global_mesh.devices.flatten()[0].platform == "gpu":
```

Comment:
Do we have a lint rule specifically saying that we need to use " instead of '? Or is this just a standard inconsistent-quotes check? https://pylint.readthedocs.io/en/latest/user_guide/checkers/features.html